### ISSUE TITLE
How to customize user profile...

### ISSUE THREAD

**Issue by bob-2165** [rustd/AspnetIdentitySample#37]
Hi,

I am having a bit of trouble with this.  I want allow the user to customize their profile with certain properties.  I want to create properties (i.e. Item A presents A, Item B presents B, etc.), which the user can then pick and choose from. So, if I only want to see Item A I can click on it and it will now be saved under my profile.  I am assuming the easiest way to do this is with a String Filter.  

Basically, how can the user have their query filter saved?
